apps/web/src/util/types.ts

```diff
@@ -4,6 +4,6 @@ export type UnwrapArray<T> = T extends (infer R)[] ? R : never;
 
 export const integrationTypeMap = new Map<IntegrationType, { image: string; name: string }>([
   [IntegrationType.TIKTOK, { name: 'TikTok', image: '/integrations/tiktok-logo-icon.svg' }],
-  [IntegrationType.FACEBOOK, { name: 'Facebook', image: '/integrations/facebook-logo-icon.svg' }],
+  [IntegrationType.FACEBOOK, { name: 'Meta', image: '/integrations/meta-logo-icon.svg' }],
```

Comment:
```suggestion
[IntegrationType.META, { name: 'Meta', image: '/integrations/meta-logo-icon.svg' }],
```

I don't think this is being used anywhere else, so it can be changed to META, too.

Reply:
this `IntegrationType` originated from a generated file, that ulitmately is traced back on the `IntegrationTypeEnum` in the database. I was a bit lazy and decided not to change it at the beginning, but your comment indicated that we still need to enforce DDD as much as possible